libcontainer/nsenter/cloned_binary.c

```diff
@@ -60,35 +60,35 @@
 
 /* Use our own wrapper for memfd_create. */
 #ifndef SYS_memfd_create
-#  ifdef __NR_memfd_create
-#    define SYS_memfd_create __NR_memfd_create
-#  else
+#ifdef __NR_memfd_create
+#define SYS_memfd_create __NR_memfd_create
+#else
 /* These values come from <https://fedora.juszkiewicz.com.pl/syscalls.html>. */
-#    warning "libc is outdated -- using hard-coded SYS_memfd_create"
```

Comment:
curious; that indentation looks like it was intentional; is there some option / other formatting that assumes those to be indented? (I'm not a c-coder, but with indentation looks somewhat more readable to me)

Reply:
I don't like most of these changes either, but don't know about any tools better than `indent` (took a quick look into `clang-format`, need to play more).

In C world it's usually a combination of CodingStyle document, a bit of help from an editor, and some common sense.

Reply:
I'm going to play a bit more with `indent` and `clang-format`

Reply:
Thanks! Glad it's not just me 😂

Reply:
OK, what I've just pushed should be somewhat better.

Reply:
Yeah the indentation was intentional, and the fixed version looks better.